transform_test.go

```diff
@@ -21,6 +21,12 @@ func TestExecFile(t *testing.T) {
 		t.Error("expected transform")
 	}
 
+	er, err := dsio.NewEntryReader(ds.Structure, body)
```

Comment:
I was tripped up by `er` and `err` being used right next to each other, assuming it was a typo on first read. How about changing to `reader` or just `r`?